frontend/src/bni-skin.css

```
@@ -0,0 +1,522 @@
+
+.bni-seg button + button {
+  border-left: 1px solid var(--bni-line2);
+}
+
+.bni-seg button.active {
+  background: var(--bni-primary);
+  color: #fff;
+}
+
+/* ---- loading skeleton ---- */
+@keyframes bniPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "bniPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)